third_party/nvidia/backend/driver.c

```diff
@@ -274,6 +274,7 @@ static PyObject *setPrintfFifoSize(PyObject *self, PyObject *args) {
   }
 
   Py_END_ALLOW_THREADS;
+  Py_INCREF(Py_None);
   return Py_None;
```

Comment:
For future reference there is a [`Py_RETURN_NONE`](https://docs.python.org/3/c-api/none.html#c.Py_RETURN_NONE) macro which does this in one line. And yes we should always increment the refcount even for constants like `Py_None` or `Py_False`.